mantidproject/mantid
# Implemented Keep Open functionality in GetNegMuMuonicXRD

The default dialog buttons have now been implemented in GetNegMuMuonicXRD. Also a bug has been fixed that occurred when Mantid was closed but the dialog was not destroyed.

**To Test**
- Load `GetNegMuMuonicXRD` algorithm
- See that the `keep open` checkbox is present as well as `run`,`cancel` and `?` buttons
- check `keep open` and run the algorithm with various inputs
- close MantidPlot without closing the dialog
- see that the dialog is closed implicitly when MantidPlot is exited.

Fixes #14379
